scss/_sjc-card.scss

```diff
@@ -1,13 +1,3 @@
-.card {
-  text-align: center;
-  box-shadow: 0 0 6px #000;
-}
-
-.card-header {
-  color: #fff;
-  background-color: $primary;
```

Comment:
>background-color: $primary;

The way to do this in bootstrap4 is to use `bg-primary` class along. No need to force all card headers to use a single color.

Reply:
As I recall, the intent was to create a St. Jude Cloud specific card, distinct from the default bootstrap card. In that case, I thing setting a default background color is fine, and still allows overriding via use of `bg-*` utility classes, as they are implemented with `!important`.

That said, I also think this particular styling should be scoped in some manner, e.g.

```scss
.sjc-card {
  .card {}
  // or
  &.card {}
}
```
or something akin to
```scss
.card--sjc { ... }
```